### ISSUE TITLE
Proposal to replace OSSP::uuid dependency for Data::UUID

### ISSUE THREAD

**Issue by hanspr** [asbru-cm/asbru-cm#374]
While looking at some old Centos problems one of the Dependencies that is usually lagging from being updated is OSSP::uuid library, compared to Data::UUID, which is always available.

We can safely replace by Data::UUID

I have made the tests, and is an extremely short change.

If you like I can send it. Or close the issue if we are fine with OSSP:uuid depencency.

**Comment by figue** [comment]
For Archlinux, Data::UUID will be better choice. Thanks!

**Comment by KlaasT** [comment]
Did you manage to have a look at Debian and Fedora if the dependencies can be resolved there? We shouldn't improve something for some distributions and make it worse for others :-)

**Comment by hanspr** [comment]
Let me rephrase : "one of the Dependencies that is usually lagging from being updated is OSSP::uuid library, compared to Data::UUID, which is always available everywhere".

Is a cpan project while OSSP:uuid is not, which makes it easy to install with cpanm

But, if we are fine with the current dependency, no problem by me, it is a proposal.

**Comment by KlaasT** [comment]
I'm fine with replacing dependencies by something new. The only thing I would like to avoid is that we have to create .deb, .rpm or any other archives to satisfy this dependency. We had to do with with the old vte libraries and bindings for gtk2 which was really annoying.

**Comment by hanspr** [comment]
I guess must be annoying, lets close the issue is not urgent or affecting many people right now.
